src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -329,9 +330,28 @@ private function getSchema(): Schema
         return $schema;
     }
 
-    public static function formatDateTime(\DateTimeInterface $dateTime)
+    /**
+     * @throws DBALException
+     *
+     * @deprecated since Symfony 4.4, use convertDateTime instead
+     */
+    public static function formatDateTime(\DateTimeInterface $dateTime, AbstractPlatform $platform = null)
+    {
+        @trigger_error(sprintf('The "%s()" method is deprecated since Symfony 4.4, use convertDateTime() instead.', __METHOD__), E_USER_DEPRECATED);
+
+        if (null === $platform) {
+            return $dateTime->format('Y-m-d\TH:i:s');
+        }
+
+        return Type::getType(Type::DATETIME)->convertToDatabaseValue($dateTime, $platform);
+    }
+
+    /**
+     * @throws DBALException
```

Comment:
should be removed. We don't care about this one here (and anyway, this API should not be public, so this would be useless)